src/ssda/ssda_populate.py

```diff
@@ -106,10 +146,6 @@ def validate_options(
             "variable FITS_BASE_DIR)."
         )
 
-    # Can not run insert for a single file
```

Comment:
This check suggests that ssda populate should not be called with the --file option. So if you remove this check (which you indeed should), you have to get rid of the --file option as well.